guides/common/modules/proc_enabling-openscap.adoc

```diff
@@ -12,3 +12,8 @@ To use the OpenSCAP plug-in and content on an external {SmartProxy}, you must en
 # {installer-scenario-smartproxy} \
 --enable-foreman-proxy-plugin-openscap
 ----
++
+Append one of the following arguments to the command:
+
+** `--foreman-proxy-plugin-openscap-ansible-module true` for Ansible deployment of compliance policies
+** `--foreman-proxy-plugin-openscap-puppet-module true` for Puppet deployment of compliance policies
```

Comment:
Should this link to the Puppet guide (at least for Katello/Satellite/Orcharhino) to note that Puppet is a prerequisite? Not to installing the module, but to actually make use of it.